google/cloud/aiplatform/base.py

```diff
@@ -974,7 +974,11 @@ def _sync_object_with_future_result(
             "_gca_resource",
             "credentials",
         ]
-        optional_sync_attributes = ["_prediction_client"]
+        optional_sync_attributes = [
+            "_prediction_client",
+            "_authorized_session",
+            "_raw_predict_request_url",
```

Comment:
These attribute names start with underscores, but the attributes in `models.py` do not have leading underscores. Is this correct?